rules_scheduler/rules_scheduler.install

```diff
@@ -107,3 +107,18 @@ function rules_scheduler_update_1200() {
     ));
   }
 }
+
+/**
+ * Install Rules Schedule view if missing.
+ */
+function rules_scheduler_update_1201() {
+  $view = views_get_view('rules_scheduler');
+  if (!$view) {
+    global $base_url;
+    $filename = $base_url . '/' . backdrop_get_path('module', 'rules_scheduler') . '/config/views.view.rules_scheduler.json';
+    $data = backdrop_json_decode(file_get_contents($filename));
+    module_load_include('inc', 'config', 'config.sync');
+    config_sync_file('views.view.rules_scheduler', 'create', $data);
+    views_invalidate_cache();
```

Comment:
I don't think it does, because `config_import_single_form_submit()` (which is where I cribbed the code from) follows `config_sync_file()` with `backdrop_flush_all_caches()`. See https://docs.backdropcms.org/api/backdrop/core%21modules%21config%21config.admin.inc/function/config_import_single_form_submit/1.

Reply:
IDK whether the update process clears all caches, but it seems to clear the Views cache, and that's the one that matters, so separate cache clear is removed.